src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/CMakeLists.txt

```diff
@@ -32,11 +32,6 @@ add_library(System.Security.Cryptography.Native.Apple-Static
     ${NATIVECRYPTO_SOURCES}
 )
 
-# Disable the "lib" prefix.
-set_target_properties(System.Security.Cryptography.Native.Apple PROPERTIES PREFIX "")
-
-# Disable the "lib" prefix and override default name
-set_target_properties(System.Security.Cryptography.Native.Apple-Static PROPERTIES PREFIX "")
```

Comment:
Same comment here, this is not only adding lib as a prefix, it is removing our override